### ISSUE TITLE
Bug / UI : Keyboard Navigation

### ISSUE THREAD

**Issue by Lewiscowles1986** [Shreerang/Vue-Nuggets#8]
Try using the `<tab>` key to navigate the example https://ecommerce-vue-nuggets.herokuapp.com/

It skips many form elements, which means for EU & US consumers the components do not allow disabled consumers to use the site and so the site could be fined.

**Comment by Shreerang** [comment]
I agree with you @Lewiscowles1986 💯 Accessibility is very important, but I haven't got to it yet. I am actively working on it, but might take some time. It is right at the top of my to do list for this project.
